com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -387,7 +399,17 @@ internal void ExecuteNativeRenderPass(ScriptableRenderContext context, Scriptabl
                     if (!AreAttachmentIndicesCompatible(m_ActiveRenderPassQueue[m_LastBeginSubpassPassIndex], m_ActiveRenderPassQueue[currentPassIndex]))
```

Comment:
this check should consider input attachments too, that way we could merge more passes potentially, and the else branch won't be needed

Reply:
since we are not sure would be practical to merge multiple subpasses with compatible input attachments for now makes sense to leave a comment explaining why the else branch is happening, and potential room for future optimization if needed (will need investigqtion)